### PR TITLE
Update gsctl info reference content

### PR DESCRIPTION
**Pull request by marians**
Updating the `gsctl info` reference page according to changes in gsctl 0.11.2

### CODE REVIEW

File: content/reference/gsctl/info.md [hunk 1]
```diff
@@ -1,46 +1,86 @@
...
-date = "2018-01-26"
+date = "2018-04-26"
 type = "page"
 weight = 41
 +++
 
 # `gsctl info`
 
-The `gsctl login` command lets you display information on your current status and configuration.
+The `gsctl login` command lets you display information on your current status
```
Comment (puja108) [inline content/reference/gsctl/info.md]: info or login?

File: content/reference/gsctl/info.md [hunk 2]
```diff
@@ -1,46 +1,86 @@
...
 
-To activate printing of the current session token, add the `-v` or `--verbose`
-flag.
 
 ## Output details
 
+- `gsctl version`: The gsctl version you are using.
+- `gsctl build`: Date and time when your copy of gsctl has been built.
+- `Config path`: Path to the currently used configuration file.
+- `kubectl config path`: Path to the kubectl configuration file in use.
```
Comment (puja108) [inline content/reference/gsctl/info.md]: does anyone call it kubectl config? isn't it just kubeconfig? I'd check what they use upstream

Comment (marians) [reply content/reference/gsctl/info.md]: I'd be willing to change this to whateber is clearly understood

Comment (puja108) [reply content/reference/gsctl/info.md]: They use `kubeconfig` or `kubeconfig file` upstream

> A file that is used to configure access to a cluster is sometimes called a `kubeconfig file`. This is a generic way of referring to configuration files. It does not mean that there is a file named `kubeconfig`.

the flag on the command line and components is also always `--kubeconfig`

but not a big issue

**Pull request merged**